common/app/views/fragments/email/stylesheets/connectedStyleOverride.scala.html

```diff
@@ -29,7 +29,8 @@
     .tone-news .headline {
         color: #333333;
     }
-    .tone-news .fc-item__kicker {
+    .tone-news .fc-item__kicker,
+    .tone-news .byline {
```

Comment:
Maybe not for this PR, but something that should really be done is moving this styling out of scala, and into scss. It would make it much easier to read, and development I am sure would be better too.